Formula/w/weaviate-cli.rb

```diff
@@ -0,0 +1,174 @@
+class WeaviateCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Command-line interface for managing and interacting with Weaviate"
+  homepage "https://pypi.org/project/weaviate-cli/"
+  url "https://files.pythonhosted.org/packages/e2/aa/d49ae3e5d399d3bb4026188017220c4dcdd240d64de88df41259056c801a/weaviate_cli-3.0.2.tar.gz"
+  sha256 "7a4a3ced248678a6d006ce849edf1d97b15237605f2735bb5ea7882b954c1fdb"
+  license "BSD-3-Clause"
+
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+  depends_on "cryptography"
+  depends_on "openssl@3"
+  depends_on "python@3.13"
```

Comment:
you also need this

```suggestion
  depends_on "python@3.13"

  uses_from_macos "libffi"
```

Reply:
Good to know, I added the `uses_from_macos "libffi"`